target/linux/ramips/dts/mt7621_hilink_hlk-7621a-evb.dts

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "hilink,hlk-7621a-evb", "mediatek,mt7621-soc";
+	model = "HiLink HLK-7621A evaluation board";
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 18 GPIO_ACTIVE_HIGH>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+};
+
+&spi0 {
+	status = "okay";
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <44000000>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "u-boot";
+				reg = <0x0 0x30000>;
+				read-only;
+			};
+
+			partition@30000 {
+				label = "u-boot-env";
+				reg = <0x30000 0x10000>;
+				read-only;
+			};
+
+			factory: partition@40000 {
+				label = "factory";
+				reg = <0x40000 0x10000>;
+				read-only;
+			};
+
+			partition@50000 {
+				compatible = "denx,uimage";
+				label = "firmware";
+				reg = <0x50000 0x1fb0000>;
+			};
+		};
+	};
+};
+
+&xhci {
+	status = "okay";
+};
```

Comment:
This can be dropped, it's enabled by default.

Reply:
Are you sure? I had to add this explicitly to be able to use the usb ports in the board. It did not working without this line.

Reply:
Maybe you tested with an older stable release?

Reply:
https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/dts/mt7621.dtsi#L371

Reply:
You are right. I have just tested with last code in master branch and works. Thanks, will drop this.